lib/datasource/terraform-provider/index.ts

```diff
@@ -220,12 +220,8 @@ export class TerraformProviderDatasource extends TerraformDatasource {
       { concurrency: 4 }
     );
 
-    // if any of the requests to build details have failed, return null
-    if (result.some((value) => Boolean(value) === false)) {
-      return null;
-    }
-
-    return result;
+    const filteredResult = result.filter(is.truthy);
```

Comment:
Is it possible to be more specific here? `!is.null_`? Or equivalent.

Reply:
It's possible, but we also want `filteredResult` to be narrowed to `TerraformBuild[]` instead of remaining just `(TerraformBuild | null)[]`

Reply:
But checking for null will do that? Doesn't matter,`is.truthy` is literally `Boolean(value)` anyway.

Reply:
If you're interested to know my rationale, checkout this out:
- https://stackoverflow.com/questions/47632622/typescript-and-filter-boolean/58110124

At first I've tried to create `util/trythy.ts` with function containing desired signature, but then just found out `is.trythy` solves the problem fine enough.